framework/build.mk

```diff
@@ -29,7 +29,7 @@ libmesh_CC       ?= $(shell METHOD=$(METHOD) $(libmesh_config) --cc)
 libmesh_F77      ?= $(shell METHOD=$(METHOD) $(libmesh_config) --fc)
 libmesh_F90      ?= $(shell METHOD=$(METHOD) $(libmesh_config) --fc)
 libmesh_INCLUDE  := $(shell METHOD=$(METHOD) $(libmesh_config) --include)
-libmesh_CPPFLAGS := $(shell METHOD=$(METHOD) $(libmesh_config) --cppflags)
+libmesh_CPPFLAGS := $(shell METHOD=$(METHOD) $(libmesh_config) --cppflags) $(ADDITIONAL_CPPFLAGS)
```

Comment:
Yeah - I realize that's not ideal either... We probably should rename all of those variables to moose_FOO and do what we want with them. That'll be a larger cleanup task in the future.